ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authorization.j2

```diff
@@ -13,6 +13,10 @@ Authorization for configuration commands is enabled.
 Authorization for configuration commands is disabled.
 {%     endif %}
 
+{%     if aaa_authorization.console is defined and aaa_authorization.console == true %}
+Authorization for serial console is enabled.
+{%     endif %}
+
```

Comment:
The blank line should be enclosed to the `if` block to not append an extra blank line when code is not rendered.

Reply:
Thanks done!